### PR TITLE
feat: list streaming providers on movie/TV detail pages

### LINKED ISSUE #654: [Feature Request] Add Provider data to Overseerr
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I'd like to see if Overseerr can utilize the existing Provider data that TMD exposes here - https://developers.themoviedb.org/3/movies/get-movie-watch-providers

That data could then be used to add a badge on top of the display of the movie/show in the GUI below where the Movie/Series badge currently displays. So showing "Netflix" or "Disney+" or "Amazon Prime" or any other providers there. Then, in addition when going into the actual item itself, the data could also be displayed on the right hand side by studio.

**Describe alternatives you've considered**
None

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

Adds list of streaming providers to movie and series detail pages.

#### Screenshot (if UI-related)

image

#### To-Dos

- [x] Successful build `yarn build`
- [x] Translation keys `yarn i18n:extract`

#### Issues Fixed or Closed

- Closes #654

### CODE REVIEW

File: src/components/TvDetails/index.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ const messages = defineMessages({
   seasons: '{seasonCount, plural, one {# Season} other {# Seasons}}',
   episodeRuntime: 'Episode Runtime',
   episodeRuntimeMinutes: '{runtime} minutes',
+  streamingproviders: 'Currently Streaming On',
```
Comment (dphildebrandt) [inline src/components/TvDetails/index.tsx:83]: I feel like “Streaming On” would be more concise? Especially if it’s an upcoming series/movie that’s not streaming yet.

Comment (dphildebrandt) [reply src/components/TvDetails/index.tsx:83]: Perhaps even “Stream Providers” or “Streaming Services” instead?

Comment (TheCatLady) [reply src/components/TvDetails/index.tsx:83]: This actually lists the _current_ streaming providers only, so I wanted to emphasize that.  Upcoming items wouldn't have any streaming providers to list.

Comment (TheCatLady) [reply src/components/TvDetails/index.tsx:83]: > Perhaps even “Stream Providers” or “Streaming Services” instead?

I originally had `Streaming Providers`, but it felt somewhat awkward so I settled on this.

Comment (dphildebrandt) [reply src/components/TvDetails/index.tsx:83]: Cool 👍

**Review by sct — APPROVED** [review]

**Pull request merged**